GNUmakefile

```diff
@@ -29,10 +29,10 @@ fmt:
 	gofmt -w $(GOFMT_FILES)
 
 fmtcheck:
-	@sh -c "'$(CURDIR)/scripts/gofmtcheck.sh'"
+	@bash -c "'$(CURDIR)/scripts/gofmtcheck.sh'"
```

Comment:
What if there's no bash?

Reply:
The respective scripts have been written assuming that bash is available. Do you suggest that the scripts be modified to run with sh ?